cli/deno_dir.rs

```diff
@@ -589,6 +622,44 @@ fn filter_shebang(bytes: Vec<u8>) -> Vec<u8> {
   }
 }
 
+fn save_code_and_headers(
+  filename: &str,
+  module_name: &str,
+  source: &str,
+  maybe_content_type: Option<String>,
+  maybe_initial_filename: Option<String>,
+) -> Result<(), DenoError> {
+  let p = PathBuf::from(filename);
+  match p.parent() {
+    Some(ref parent) => fs::create_dir_all(parent),
+    None => Ok(()),
+  }?;
+  // Write file and create .headers.json for the file.
+  deno_fs::write_file(&p, &source, 0o666)?;
+  {
+    save_source_code_headers(filename, maybe_content_type.clone(), None);
```

Comment:
`save_code_and_headers` and `save_source_code_headers` are too similarly named. It took me a minute of staring at this to before I realized it wasn't recursive. Please choose new names that are more distinct.

Reply:
> save_code_and_headers and save_source_code_headers are too similarly named.

Agreed, will rename